internal/scripts/installation/modulefiles/5.0.0_casper

```diff
@@ -0,0 +1,18 @@
+#%Module######################################################################
+##
+##      METplus
+##
+proc ModulesHelp { } {
+  puts stderr "Sets up the paths and environment variables to use the METplus-4.1.1.
```

Comment:
Incorrect version number. Should be 5.0.0 instead of 4.1.1

Reply:
Great catch!  Thank you!  I just updated the file in the repo and on casper.  I'll go over to the main PR and do the same in that one.